README.md

```diff
@@ -0,0 +1,101 @@
+Spryker Buggregator
+=============
+
+Provide integration of [buggregator](https://buggregator.dev/) service with Spryker.
+
+Installation
+------------
+
+Require the package with composer:
+
+```
+$ composer require teufelaudio/spryker-buggregator
+```
+
+Add the `Teufelaudio` namespace to the `CORE_NAMESPACES` key in your `config/Shared/config_default.php` file:
+
+```php
+$config[KernelConstants::CORE_NAMESPACES] = [
+    // ...
+    'Teufelaudio',
+];
+```
+
+Register `RequestProfilerEventDispatcherPlugin` for Yves:
+
+```php
+<?php
+
+namespace Teufel\Yves\EventDispatcher;
+
+use Spryker\Yves\EventDispatcher\EventDispatcherDependencyProvider as SprykerEventDispatcherDependencyProvider;
+use Teufelaudio\Yves\SprykerBuggregator\Plugin\EventDispatcher\RequestProfilerEventDispatcherPlugin;
+
+class EventDispatcherDependencyProvider extends SprykerEventDispatcherDependencyProvider
+{
+    protected function getEventDispatcherPlugins(): array
+    {
+        return [
+            // ...
+            new RequestProfilerEventDispatcherPlugin(),
+        ];
+    }
+} 
+```
+
+Register `ConsoleProfilerListener` for Zed:
+
+```php
+<?php
+
+namespace Teufel\Zed\Console;
+
+use Spryker\Zed\Console\ConsoleDependencyProvider as SprykerConsoleDependencyProvider;
+use Teufelaudio\Zed\SprykerBuggregator\Communication\Console\EventListener\ConsoleProfilerListener;
+
+class ConsoleDependencyProvider extends SprykerConsoleDependencyProvider
+{
+    public function getEventSubscriber(Container $container): array
+    {
+        return [
+            // ...
+            new ConsoleProfilerListener(),
+        ];
+    
+    }
+}
+```
+
+Register the `RequestProfilerEventDispatcherPlugin` for Zed:
+
+```php
+<?php
+
+namespace Teufel\Zed\EventDispatcher;
+
+use Spryker\Zed\EventDispatcher\EventDispatcherDependencyProvider as SprykerEventDispatcherDependencyProvider;
+use Teufelaudio\Zed\SprykerBuggregator\Communication\Plugin\EventDispatcher\RequestProfilerEventDispatcherPlugin;
+
+class EventDispatcherDependencyProvider extends SprykerEventDispatcherDependencyProvider
+{
+    protected function getEventDispatcherPlugins(): array
+    {
+        return [
+            // ...
+            new RequestProfilerEventDispatcherPlugin(),
+        ];
+    }
+}
+```
+
+Add following configuration to your `config_default-development.php`:
+
+```php
+use Teufelaudio\Shared\SprykerBuggregator\SprykerBuggregatorConstants;
```

Comment:
I'm wondering if we need the "Spryker" prefix for the module. "teufelaudio/spryker-buggregator" feels correct for the composer package, because it indicates it is a composer package that connects Spryker with Buggregator. But when we are looking internally in Spryker, the prefix could be removed.

Reply:
I'm thinking this from the "user" perspective, if I'm importing a class like `use Teufelaudio\Shared\Buggregator\BuggregatorConstants`, it does not make it clear that this class is related to Spryker's buggregator integration. I know its there in the package name, but I would say we should keep it in module name too.